build-presets/MakePresetTree.py

```diff
@@ -78,9 +78,9 @@
 
 
 # %% Save as javascript files
-with open('preset_tree.min.js', 'w') as file:
+with open('../js/preset_tree.min.js', 'w') as file:
     file.write('var preset_tree = ' + json.dumps(preset_tree, separators=(',', ':')))
 # Create preset_tree.js if you want a version that is more legible to humans    
```

Comment:
I'd suggest we uncomment this by default, so the human version aligns with the `.min` version. What do you think?

Reply:
Ah I see your point. Tim had requested I comment it out, but I think I agree with you